darts/models/forecasting/pl_forecasting_module.py

```diff
@@ -30,6 +30,7 @@ def __init__(
         self,
         input_chunk_length: int,
         output_chunk_length: int,
+        train_sample_shape: Tuple = None,
```

Comment:
Missing parameter definition and
```suggestion
        train_sample_shape: Optional[Tuple] = None,
```